### ISSUE TITLE
Bug with Ember 2.17, IE11

### ISSUE THREAD

**Issue by chriskrycho** [peec/ember-css-transitions#23]
https://github.com/peec/ember-css-transitions/blob/e1a3dd81f9cce238f078de29858bb0ff206ee8f7/addon/mixins/transition-mixin.js#L217-L224

This bit of code assumes one of two things is true:

1. Per the comment, that `this.element` further back up the chain is a jQuery item (in our app, it's not!) or…
2. That it's running on a browser that isn't IE11, since everything else has supported `Node.remove` since 2014.

This gave us a bug in production today (thankfully, affecting only a small number of users).

Our workaround was to polyfill it [as suggested by MDN](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove#Polyfill). Shipping the polyfill (possibly as an opt-in for people who need to target IE11?) seems like it might be the best solution?

**Comment by miguelcobain** [comment]
An unfortunate situation. 😞 
Thanks for the report.

I can think of a number of solutions:

1. conditionally ship the polyfill with this addon according to a config option. We should probably include the polyfill by default since ember supports IE11.
2. use ember-cli targets to determine if the polyfill needs to be included and include it if needed
3. do the same as 2 but in an external "ember-maybe-import-remove-polyfill" addon which this addon would then depend on

However, on a second thought, I think all of this is overkill because we could just avoid using `remove()` by using:
```diff
didTransitionOut() { 
-  this.clone.remove();
+  this.clone.parentNode.removeChild(this.clone); 
} 
```

What do you think? Do you have time for such a PR?

**Comment by miguelcobain** [comment]
Solved by #24

**Comment by kernel-io** [comment]
I hope this wasn't my fault :cry:
